libpod/pod.go

```diff
@@ -97,6 +98,7 @@ type InfraContainerConfig struct {
 	HasInfraContainer  bool                  `json:"makeInfraContainer"`
 	NoNetwork          bool                  `json:"noNetwork,omitempty"`
 	HostNetwork        bool                  `json:"infraHostNetwork,omitempty"`
+	Pid                specgen.Namespace     `json:"infraPid,omitempty"`
```

Comment:
I think this should be called `PidNS` to prevent mistaking it for a process ID.  Same at other places in the code.

But I do agree that `--pid` makes sense on the CLI to remain consistent with `container create`.